invenio_files_rest/config.py

```diff
@@ -100,3 +107,9 @@
 
 FILES_REST_MULTIPART_EXPIRES = timedelta(days=4)
 """Time delta after which a multipart upload is considered expired."""
+
+FILES_REST_TASK_WAIT_INTERVAL = 2.0
```

Comment:
I also like uniformity, but in this context I don't think it makes sense to use ``timedelta``. I associate ``timedelta``  with in date/datetime calculations, not with short timeouts.